src/webui/www/private/transferlist.html

```diff
@@ -42,17 +42,17 @@
             Rename: function(element, ref) {
                 renameFN();
             },
-            prioTop: function(element, ref) {
-                setPriorityFN('topPrio');
+            QueueTop: function(element, ref) {
```

Comment:
Why it starts with Capital letter?

Reply:
I followed the style of the other ones.

Reply:
However, this style does not follow the style of the project... Okay, whatever. I don't care.

Reply:
Doesn't sound like you don't care but there's not much i can do about it because i can't even tell which ones are the vars.

Reply:
You should use lowercase for new code.

Reply:
Added new commit to rename some of them.